controllers/ratelimitpolicy_enforced_status_controller.go

```diff
@@ -200,6 +200,8 @@ func (r *RateLimitPolicyEnforcedStatusReconciler) buildTopology(ctx context.Cont
 	}
 
 	return kuadrantgatewayapi.NewTopology(
+		kuadrantgatewayapi.WithAcceptedRoutesLinkedOnly(),
+		kuadrantgatewayapi.WithProgrammedGatewaysOnly(true),
```

Comment:
I understand a gateway that transitions from `Programmed: True` to `Programmed: False` will NOT get the statuses of its affecting policies updated then?

I'm thinking this line invoked for a gateway that has not been added to the topology: https://github.com/Kuadrant/kuadrant-operator/blob/4b9d60e49eae89460a963b9395f1e92a8f837332/controllers/ratelimitpolicy_enforced_status_controller.go#L60

Reply:
good catch, thanks!

Reply:
fixed